### ISSUE TITLE
prediction result is unconsistent

### ISSUE THREAD

**Issue by aaronlyt** [awslabs/djl#149]
## Description

using pytorch as the backend model, the same input, we run 100 time, but get different result? 

in the pytorch model, contain **Linear layer and BatchNorm Layer**, Does BatchNorm Layer need more processing?

**Comment by lanking520** [comment]
Please let us know if you have any script to reproduce. We will take a look
